tests/e2e/keadm/keadm_test.go

```diff
@@ -47,44 +52,27 @@ var _ = Describe("Application deployment test in keadm E2E scenario", func() {
 			testTimer.End()
 			// Print result
 			testTimer.PrintResult()
-			var podlist corev1.PodList
-			label := nodeName
-			podlist, err := utils.GetPods(ctx.Cfg.K8SMasterForKubeEdge+constants.AppHandler, label)
+
+			labelSelector := labels.SelectorFromSet(map[string]string{"kubeedge": "e2e-test"})
```

Comment:
i found a new label: "e2e-test", is this designed?
almost all tests can be done with "app": "podname"

Reply:
for pod test, we may create multi pod and we use this label for the pod that is created by e2e and after each case finished, we use the label to clean the pod.